### DISCUSSION TITLE
NCA analysis using PKNCA - Back extrapolation not possible

### DISCUSSION THREAD

**Discussion by caribap** [humanpred/pknca#294] [Q&A] [answered]
Rversion = 4.3.2
I want to perform an NCA analysis for a single dose studies. I have concentration measurements at 5 timepoints from 0.25 to 24h for each subject, and have 3 subjects for each single dose - 1) 1mg/kg IV, 2) 10mg/kg IV and 3) 10mg/kg PO. In the end I want to have a table with pk parameters for the mean between the 3 subjects at each single dose study.

My problem is that PKNCA is not able to understand that I don't have time points before 0.25; in phoenix this is automatically done by back extrapolation. In PKNCA I get different results or no results at all - cant get an AUClast value right.

Note: I am using asnumeric and is.na to make sure my dose, concentration and time are manipulated correctly; I am using read_excel() to read my data and in the beginning I split my data by route creating 2 data frames

Thank you.

**Comment by billdenney** [comment, marked as answer]
Please check out the data imputation vignette here for how to do the initial imputation: https://billdenney.github.io/pknca/articles/v08-data-imputation.html#imputation-for-the-full-dataset

That has code examples that will hopefully clear it up for you.
